src/modules/utils.js

```diff
@@ -567,3 +567,56 @@ export const isEdgeLogin = (data: string) => {
 
   return EDGE_LOGIN_REG_EXP.test(data)
 }
+
+export const tallyUpTotalCrypto = (state: State) => {
```

Comment:
how expensive is this function?

Reply:
Not very, in the low 10's of milliseconds.